### ISSUE TITLE
Bad state: No scripts found error

### ISSUE THREAD

**Issue by JWambaugh** [chenasraf/dart_script_runner#2]
Hello! Thanks for the great tool. I've found however that after updating to 0.6.0, I get this error trying to run anything:
```
scr --help                
Bad state: No scripts found
#0      runScript (package:script_runner/src/base.dart:7:5)
<asynchronous suspension>
#1      main (file:///[redacted]/.pub-cache/hosted/pub.dev/script_runner-0.6.0/bin/script_runner.dart:14:5)
<asynchronous suspension>
```
I am setting the config in my pubspec.yaml file, and it was working fine before 0.6.0.

**Comment by chenasraf** [comment]
I'm sorry about that! Thanks for the report.

I will look into it and will update you when I push a solution.

**Comment by chenasraf** [comment]
Hi @JWambaugh, I believe it should be fixed in v0.6.1 (just released).

Please re-open the issue if it still isn't resolved for you.

Thanks again for reporting the issue!

**Comment by JWambaugh** [comment]
It's working great now. Thanks!
